desktop/renderer/index.js

```diff
@@ -124,4 +124,4 @@ class Keybase extends Component {
   }
 }
 
-ReactDOM.render(<Keybase/>, document.getElementById('app'))
+ReactDOM.render(<Keybase />, document.getElementById('app'))
```

Comment:
i don't really care. this comes from standard. i want it one way or the other, right now its a mix